stellar-contract-env-host/src/host.rs

```diff
@@ -1704,18 +1724,26 @@ impl CheckedEnv for Host {
     }
 
     fn account_get_low_threshold(&self, a: Object) -> Result<RawVal, Self::Error> {
-        todo!()
+        Ok(self.load_account(a)?.thresholds.0[ThresholdIndexes::Low as usize].into())
     }
 
     fn account_get_medium_threshold(&self, a: Object) -> Result<RawVal, Self::Error> {
-        todo!()
+        Ok(self.load_account(a)?.thresholds.0[ThresholdIndexes::Low as usize].into())
     }
 
     fn account_get_high_threshold(&self, a: Object) -> Result<RawVal, Self::Error> {
-        todo!()
+        Ok(self.load_account(a)?.thresholds.0[ThresholdIndexes::Low as usize].into())
```

Comment:
Should be `ThresholdIndexes::High`.

Reply:
Copy paste error during refactoring. Good eye, fixed.